.cirrus.yml

```diff
@@ -81,6 +105,7 @@ qa_os_win_task:
   cleanup_before_cache_script: cleanup_maven_repository
 
 plugin_qa_task:
+  only_if: $CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*"
```

Comment:
why this addition?

Reply:
Good catch. That line should not be there. The change should now be reverted.